schema/divisions/division.yaml

```diff
@@ -0,0 +1,178 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: division
+description:
+  Divisions are recognized official or non-official organizations of
+  people as seen from a given political perspective. Examples include
+  countries, provinces, cities, towns, neighborhoods, etc.
+type: object
+properties:     # JSON Schema: Top-level object properties.
+  id: { "$ref": ../defs.yaml#/$defs/propertyDefinitions/id }
+  geometry:
+    description:
+      Division geometry MUST be a Point as defined by GeoJSON schema.
+      It represents the approximate location of a position commonly
+      associated with the real-world entity modeled by the division
+      feature.
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": https://geojson.org/schema/Point.json
+  properties:   # GeoJSON: top-level object 'properties' property.
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/namesContainer
+      - if:
+          properties:
+            subtype: { enum: [country] }
+        then:
+          required: [names, subtype, country, hierarchies]
+          not: { required: [parent_division_id] }
+        else:
+          required: [names, subtype, country, hierarchies, parent_division_id]
+    properties: # JSON Schema: properties within GeoJSON top-level object 'properties' property
+      subtype: { "$ref": "./defs.yaml#/$defs/typeDefinitions/placetype" }
+      local_type:
+        description:
+          Local name for the subtype property, optionally localized.
+
+          For example, the Canadian province of Quebec has the subtype
+          'region', but in the local administrative hierarchy it is
+          referred to as a 'province'. Similarly, the Canadian Yukon
+          territory also has subtype 'region', but is locally called
+          a 'territory'.
+
+          This property is localized using a standard Overture names
+          structure. So for example, in Switzerland the top-level
+          administrative subdivision corresponding to subtype 'region'
+          is the canton, which is may be translated in each of
+          Switzerland's official languages as, 'canton' in French, 
+          'kanton' in German, 'cantone' in Italian, and 'chantun' in
+          Romansh.
+        allOf:
+          - "$ref": "../defs.yaml#/$defs/propertyDefinitions/commonNames"
```

Comment:
It's nice to have this map but I wonder how necessary it is... and how well we'll be able to populate it but I like the consistency with the names structure

Reply:
Yes, I don't really know if we will be able to. I'll take that up with the admins group. If they think we can never get data for it, it seems like legitimate grounds for deletion.

Reply:
Added to the agenda for Monday's meeting ([2024-03-04](https://github.com/OvertureMaps/tf-admin/issues/51)).

Reply:
The outcome of the meeting was that we will merge the PR as-is and I will follow fast with a smaller that removes the `primary` property, so there is just the names map.

This will resolve the difficult conundrum of what to set as "primary" in a multi-lingual country, by simply removing it.